### DISCUSSION TITLE
You can still query metrics data even if the OpenTelemetry client no longer sends that metrics data

### DISCUSSION THREAD

**Discussion by gaochuang** [open-telemetry/community#1540] [Community Discussions]
Deployment:
opentelemetry client(pod) + opentelmetry collector + prometheus 

Function:
Send the CPU usage of multiple processes to the OpenTelemetry Collector through the OpenTelemetry client metrics.

Issue:
When a process stops running, the OpenTelemetry client will not send the CPU usage of that process. However, the OpenTelemetry collector will always query the CPU usage rate of processes, even if they have stopped

**Comment by gaochuang** [comment]
I didn't find relevant information, and I don't know how to configure opentelemetry collector.

**Reply by trask** [reply]
hi @gaochuang! I'd recommend asking in https://github.com/open-telemetry/opentelemetry-collector-contrib or [`#otel-collector`](https://cloud-native.slack.com/archives/C01N6P7KR6W) slack channel

**Reply by gaochuang** [reply]
@trask  Hi, Task, Thanks for your help.
